webpack.config.js

```diff
@@ -1,31 +1,36 @@
 const path = require('path');
-const nodeExternals = require('webpack-node-externals');
 
-const baseConfig = require('./webpack.base.config');
-
-const webConfig = Object.assign({}, baseConfig, {
-  target: 'web',
+const webConfig =  {
+  entry: './build/src/index.js',
   mode: "production",
+  target: 'web',
+  module: {
```

Comment:
Do we still need it since we are building from JS?

Reply:
Yes for the target:web (obviously). 
You are right that the loader might be too much, will fix it !